packages/strapi/lib/middlewares/public/index.js

```diff
@@ -106,7 +103,7 @@ module.exports = strapi => {
           await next();
         },
         koaStatic(buildDir, {
-          index: indexFileName,
+          index: '/',
```

Comment:
This serves the /admin route with alwasy have an `index.html` you can leave this unchanged ;)